### ISSUE TITLE
Desert theme supposed to have dark background?

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alqxyz/google-code-prettify#255]
```
I just viewed the gallery page at:

http://google-code-prettify.googlecode.com/svn/trunk/styles/index.html

and noticed the desert theme seems wrong.  Attached screenshot.

Using Chrome, up-to-date.
```

Original issue reported on code.google.com by `alley.ry...@gmail.com` on 1 Nov 2012 at 1:10

Attachments:
- [gcp_issue.JPG](https://storage.googleapis.com/google-code-attachments/google-code-prettify/issue-255/comment-0/gcp_issue.JPG)

**Comment by GoogleCodeExporter** [comment]
```
Thank you.  The theme included background: #333 when not in media=print but was 
ignored because the class selector was wrong.  Fixed in r227.   This will be 
included in a forthcoming release.
```

Original comment by `mikesamuel@gmail.com` on 4 Feb 2013 at 9:55
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
nj buddy
```

Original comment by `alley.ry...@gmail.com` on 4 Feb 2013 at 9:56
